### ISSUE TITLE
Questions about the algorithm

### ISSUE THREAD

**Issue by MYM181** [gbliao/SPC-GS#2]
Hi, I'm interested in your work.
I'm wondering that if the semantic losses will contribute to the optimization of all attributes of 3DGS(including location, opacity...) or just for the optimization of semantic features?
How to understand the contribution of semantic regularizations to novel view rendering? 
Maybe the improvement on novel view rendering is mainly due to the SGI (mainly because of better initialization than SfM)?

**Comment by gbliao** [comment]
Hi, thanks for your interest in our work.

1) Yes, the semantic losses do contribute to the optimization of all attributes of 3DGS, including location and opacity, through gradient influence.

2) Semantic regularizations help to constrain the overlapping regions between novel views and training views, enhancing multi-view consistency. 

3) SGI also plays a significant role in improving novel view rendering.

**Comment by MYM181** [comment]
> Hi, thanks for your interest in our work.
> 
> 1. Yes, the semantic losses do contribute to the optimization of all attributes of 3DGS, including location and opacity, through gradient influence.
> 2. Semantic regularizations help to constrain the overlapping regions between novel views and training views, enhancing multi-view consistency.
> 3. SGI also plays a significant role in improving novel view rendering.

Thanks for your answer.
Sure, I understand the semantic regularization enhances multi-view consistency. The multi-view consistency of semantic features is easy to understand.
But how does it influence the attributes of 3DGS (such as opacity and locations), that can result in better novel view rendering performance but not worse? Is there any detailed analysis on it?
Wish we can have further discussions about it. Thanks.

**Comment by gbliao** [comment]
Hi. The **gradients** from the semantic constraint loss function affect the attributes of 3DGS. Especially in sparse-view input conditions, the 'consistency' constraints from novel views help alleviate the under-constrained problem. 

I hope that this explanation is helpful.
